cmd/nginx-ingress/flags.go

```diff
@@ -201,6 +203,10 @@ var (
 
 	enableDynamicSSLReload = flag.Bool(dynamicSSLReloadParam, true, "Enable reloading of SSL Certificates without restarting the NGINX process.")
 
+	enableTelemetryReporting = flag.Bool("enable-telemetry-reporting", true, "Enable gathering and reporting of product related telemetry.")
```

Comment:
Do we have a task for controlling this via helm or should it be added as part of this PR?

Reply:
We don't have helm-related tasks atm. We would need to add this to Helm before releasing v3.5.

Reply:
Thanks @oseoin nice catch. I'll make this change as part of this PR.

Reply:
Task is also added to the list.